lib/DDG/Spice/RiffsyGifSearch.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::RiffsyGifSearch;
+# ABSTRACT: Search Riffsy (http://www.riffsy.com) for animated GIFs.
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "RiffsyGifSearch";
+source "Riffsy";
+description "Search Riffsy for animated GIFs";
+primary_example_queries "riffsy lol", "cat riffs", "dog gifs";
+category "entertainment";
+topics "everyday", "entertainment", "food_and_drink", "gaming", "geek", "music", "science", "social", "special_interest";
+code_url "https://github.com/bryanhart/zeroclickinfo-spice/blob/master/lib/DDG/Spice/RiffsyGifSearch.pm";
+attribution web => ["http://www.riffsy.com","Riffsy"],
+            twitter => "riffsyapp";
+
+
+spice to => 'http://api1.riffsy.com/v1/search?tag=$1&key={{ENV{DDG_SPICE_RIFFSY_APIKEY}}}&safesearch=strict';
+spice wrap_jsonp_callback => 1;
+
+triggers any => "riffsy", "riff", "riffs", "gif", "gifs";
```

Comment:
We can add these terms to our regex above. Something like this would work:

```perl
triggers query_lc => qr/^\X$|\b(\X|[\w\s]+?) (riff(sy?)?|gifs?)\b/;
```
More info: https://www.debuggex.com/r/bC59FesFC2Aq3gDX